google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/resources/ProjectSelector.java

```diff
@@ -434,7 +437,7 @@ void setFilter(String filter) {
           GoogleUserModelItem userModelItem = (GoogleUserModelItem) path.getLastPathComponent();
           userModelItem.setFilter(filter);
           DefaultTreeModel model = (DefaultTreeModel) tree.getModel();
-          model.reload(userModelItem);
+          model.nodeStructureChanged(userModelItem);
```

Comment:
does `nodeStructureChanged` have a different visible outcome from `reload`?

Reply:
Not visible but from what I could tell, it performed better than `reload()`, probably because it lets one of the listeners do the reloading instead of forcing it here.